ios/FluentUI/Table View/TableViewCellFileAccessoryView.swift

```diff
@@ -380,6 +380,13 @@ private class FileAccessoryViewActionView: UIButton {
             widthAnchor.constraint(equalToConstant: FileAccessoryViewActionView.size.width),
             heightAnchor.constraint(greaterThanOrEqualToConstant: FileAccessoryViewActionView.size.height)
         ])
+
+        if #available(iOS 13, *) {
+            addInteraction(UILargeContentViewerInteraction())
```

Comment:
shouldn't this be in the container view of all buttons and the button just need to set the scale image and title? https://developer.apple.com/videos/play/wwdc2019/261/